packages/back-end/src/services/athena.ts

```diff
@@ -1,20 +1,52 @@
-import { Athena } from "aws-sdk";
-import { ResultSet } from "aws-sdk/clients/athena";
+import { STSClient, AssumeRoleCommand } from "@aws-sdk/client-sts";
+import { Athena, ResultSet } from "@aws-sdk/client-athena";
 import { AthenaConnectionParams } from "../../types/integrations/athena";
 import { logger } from "../util/logger";
 import { IS_CLOUD } from "../util/secrets";
 import { ExternalIdCallback, QueryResponse } from "../types/Integration";
 
-function getAthenaInstance(params: AthenaConnectionParams) {
+async function assumeRole(params: AthenaConnectionParams) {
+  // build sts client
+  const client = new STSClient({ region: "us-east-1" });
```

Comment:
Should this come from `params.region` instead of being hard-coded?

Reply:
Ah, so with STS it's a global service but I wasn't sure if the client needs a region supplied, ideally not.

Since Athena can be in different regions it makes sense to feed that in as a connection parameter during setup.

Reply:
dropped the region as it's not required